### PR TITLE
Render changelog in the website

### PR DESCRIPTION
**Pull request by xtuc**
Trying to solve #1037 

changelog

### TODO

- [x] Emoji are not supported
- [ ] Codeblocks are broken

### CODE REVIEW

**Comment by hzoo** [comment]
yeah fixes would probably have to go in CHANGELOG.md itself

**Comment by xtuc** [comment]
Since we want to include the README in babel/babel, I think it's not a good idea to store picture in inside the repo.
We could maybe use a script to replace it dynamically on the website ?

**Comment by hzoo** [comment]
what do you mean by pictures? emoji should render correctly on the readme as well

**Comment by xtuc** [comment]
I meant replacing the emoji by actual pictures. So this could be displayed on both website and github (and others). But since we should store them in babel/babel, this is not a good idea.

I suggest replacing them using JavaScript on the client-side of the website.

**Comment by xtuc** [comment]
Since #1074 we can use emoji on the website.
I guess the CHANGELOG is generated, I'm not sure how to fix the code blocks.

/cc @existentialism

**Comment by hzoo** [comment]
Change log isn't generated - it's first generated and then I modify it so we can make changes

**Comment by xtuc** [comment]
changelog

Code blocks are not broken.

File: docs/changelog.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: docs
+title: CHANGELOG
```
Comment (existentialism) [inline docs/changelog.md]: Should this be all caps?

**Comment by existentialism** [comment]
_Some_ code blocks are:

image

**Comment by existentialism** [comment]
We might have to re-visit the sidebar generation code again since it looks like scrollSpy/sidebar is broken on this page

File: docs/changelog.md (hunk 1 above)
Comment (hzoo) [inline docs/changelog.md]: We can just do Changelog

**Comment by xtuc** [comment]
I tried to add a scrollbar to the sidebar. This is not working well... I guess we will need to create a separate issue for this.

scrollbar-changelog

**Comment by xtuc** [comment]
@existentialism Ok for merging this first ?

**Comment by existentialism** [comment]
Should we fix the code blocks that aren't working (likely need newlines) first?

image

**Comment by existentialism** [comment]
Also the `CHANGELOG - 6to5` is broken

**Pull request closed without merging**